library/data/loot_table/src/testmod/java/org/quiltmc/qsl/loot/test/LootTableTestMod.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019 FabricMC
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.loot.test;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSerializationContext;
+import net.fabricmc.api.ModInitializer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import net.minecraft.item.Items;
+import net.minecraft.loot.LootGsons;
+import net.minecraft.loot.LootPool;
+import net.minecraft.loot.condition.LootCondition;
+import net.minecraft.loot.condition.SurvivesExplosionLootCondition;
+import net.minecraft.loot.entry.ItemEntry;
+import net.minecraft.loot.entry.LootPoolEntry;
+import net.minecraft.loot.entry.LootPoolEntryType;
+import net.minecraft.loot.entry.TagEntry;
+import net.minecraft.loot.provider.number.ConstantLootNumberProvider;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.JsonHelper;
+import net.minecraft.util.registry.Registry;
+
+import org.quiltmc.qsl.loot.api.QuiltLootPoolBuilder;
+import org.quiltmc.qsl.loot.api.QuiltLootTableBuilder;
+import org.quiltmc.qsl.loot.api.event.LootTableLoadingCallback;
+
+public class LootTableTestMod implements ModInitializer {
+	private static final Logger LOGGER = LogManager.getLogger();
+
+	private static final Gson LOOT_GSON = LootGsons.getTableGsonBuilder().create();
+	private static final String LOOT_ENTRY_JSON = "{\"type\":\"minecraft:item\",\"name\":\"minecraft:apple\"}";
+
+	@Override
+	public void onInitialize() {
+		// Test loot entry
+		Registry.register(Registry.LOOT_POOL_ENTRY_TYPE, new Identifier("quilt", "extended_tag"), new LootPoolEntryType(new TestSerializer()));
```

Comment:
There's no need to test this anymore, I suppose, since there's no custom API use here :smile:

Reply:
I replaced the test with yours, is that fine?

Reply:
Yep, that's totally fine.